crates/aptos-telemetry-service/src/tests/mod.rs

```diff
@@ -0,0 +1,20 @@
+mod test_context;
+mod auth_test;
+
+/// Returns the name of the current function. This macro is used to derive the name for the golden
+/// file of each test case.
+#[macro_export]
+macro_rules! current_function_name {
```

Comment:
let's not copy and paste this but rather put this into some central utility crate/library and import it from there.